### DISCUSSION TITLE
Moss can eb used to grief a claim - setup issue or bug?

### DISCUSSION THREAD

**Discussion by christina-kline** [GriefPrevention/GriefPrevention#2376] [Q&A] [answered]
As said, apparently on my server moss can be used to change a claim. Is this a bug, is it known, or is there a setting I'm missing?

**Comment by christina-kline** [comment]
I did find this issue: https://github.com/GriefPrevention/GriefPrevention/issues/1432
The issue is closed, and the last note on it is "well, this is in master, but not in production ready builds". Is there a production ready build that this is included in or a date for it's implementation?

**Reply by Jikoo** [reply, marked as answer]
No, there is no release that has been cut that contains #1921. Additionally, as it was an excessively large PR (meaning it also took a correspondingly large amount of time to test and then be approved), I don't personally have any interest in backporting it to the legacy branch.

In my opinion anything you can test and verify the functionality of is production ready, so v17 snapshots are fine to use in a live environment once you've tried them out on a test server, assuming you don't want any of the features that were cut from v16. You can find a full list of changes in the [commits since the creation of the v17.0.0 tag](https://github.com/GriefPrevention/GriefPrevention/compare/17.0.0...master). Some cut features may also be available via addons.

If v17 isn't to your liking, I'd recommend telling players to claim more than the bare minimum so it becomes less of an issue. Griefers will never convert my dirt box (I am atrocious at building) into moss because I claimed a nice wide radius around it 😎

**Reply by christina-kline** [reply]
Thank you for the info! I'm a newer admin, and the only admin on a server with a lot of claims already, so even the idea of updating and messing those claims up is scary. I'll take the last approach there - just claim more lol
